<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>includeExcludeOptions</class>
 <widget class="QWidget" name="includeExcludeOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>761</width>
        <height>654</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QCheckBox" name="dontScanFileNameCheckBox">
         <property name="text">
          <string>Don't scan file names matching REGEX</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="dontScanFileNameLineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="dontScanDirCheckBox">
         <property name="text">
          <string>Don't scan directories matching REGEX</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="dontScanDirLineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="onlyScanFileNameCheckBox">
         <property name="text">
          <string>Only scan file names matching REGEX</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="onlyScanFileNameLineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="onlyScanDirCheckBox">
         <property name="text">
          <string>Only scan directories matching REGEX</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="onlyScanDirLineEdit"/>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::Panel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QCheckBox" name="enablePUACheckBox">
            <property name="text">
             <string>Use Signatures for Potentially Unwanted Applications</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QFrame" name="PUAFrame">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QRadioButton" name="loadPUAPackedRadioButon">
               <property name="text">
                <string>Load PUA sigs of category Packed</string>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="7" column="0">
              <widget class="QRadioButton" name="loadPUANetToolSpyRadioButton">
               <property name="text">
                <string>Load PUA sigs of category NetToolSpy</string>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="8" column="0">
              <widget class="QRadioButton" name="loadPUAServerRadioButton">
               <property name="text">
                <string>Load PUA sigs of category Server</string>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QRadioButton" name="loadPUAPWToolRadioButton">
               <property name="text">
                <string>Load PUA sigs of category PWTool</string>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QRadioButton" name="loadPUARATRadioButton">
               <property name="text">
                <string>Load PUA sigs of category RAT</string>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QRadioButton" name="loadPUANetToolRadioButton">
               <property name="text">
                <string>Load PUA sigs of category NetTool</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QRadioButton" name="loadPUAScriptRadioButton">
               <property name="text">
                <string>Load PUA sigs of category Script</string>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QRadioButton" name="loadPUAIRCRadioButton">
               <property name="text">
                <string>Load PUA sigs of category IRC</string>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QRadioButton" name="loadPUAP2PRadioButton">
               <property name="text">
                <string>Load PUA sigs of category P2P</string>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QRadioButton" name="loadPUAAndrRadioButton">
               <property name="text">
                <string>Load PUA sigs of category Andr</string>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QRadioButton" name="loadPUAJavaRadioButton">
               <property name="text">
                <string>Load PUA sigs of category Java</string>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QRadioButton" name="loadPUAOsxRadioButton">
               <property name="text">
                <string>Load PUA sigs of category Osx</string>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QRadioButton" name="loadPUAToolRadioButton">
               <property name="text">
                <string>Load PUA sigs of category Tool</string>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QRadioButton" name="loadPUAUnixRadioButton">
               <property name="text">
                <string>Load PUA sigs of category Unix</string>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="7" column="1">
              <widget class="QRadioButton" name="loadPUAWinRadioButton">
               <property name="text">
                <string>Load PUA sigs of category Win</string>
               </property>
               <property name="autoExclusive">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>enablePUACheckBox</sender>
   <signal>clicked()</signal>
   <receiver>includeExcludeOptions</receiver>
   <slot>slot_enablePUACheckBoxClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>33</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dontScanFileNameLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>includeExcludeOptions</receiver>
   <slot>slot_writeSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>600</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>662</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dontScanDirLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>includeExcludeOptions</receiver>
   <slot>slot_writeSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>523</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>665</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>onlyScanFileNameLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>includeExcludeOptions</receiver>
   <slot>slot_writeSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>529</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>666</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>onlyScanDirLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>includeExcludeOptions</receiver>
   <slot>slot_writeSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>541</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>665</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadPUAPackedRadioButon</sender>
   <signal>clicked()</signal>
   <receiver>includeExcludeOptions</receiver>
   <slot>slot_writeSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>665</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadPUAPWToolRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>includeExcludeOptions</receiver>
   <slot>slot_writeSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>663</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadPUANetToolRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>includeExcludeOptions</receiver>
   <slot>slot_writeSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>346</y>
    </hint>
    <hint type="destinationlabel">
     <x>666</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadPUAP2PRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>includeExcludeOptions</receiver>
   <slot>slot_writeSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>664</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadPUAIRCRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>includeExcludeOptions</receiver>
   <slot>slot_writeSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel">
     <x>664</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadPUARATRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>includeExcludeOptions</receiver>
   <slot>slot_writeSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>502</y>
    </hint>
    <hint type="destinationlabel">
     <x>666</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadPUANetToolSpyRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>includeExcludeOptions</receiver>
   <slot>slot_writeSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>554</y>
    </hint>
    <hint type="destinationlabel">
     <x>666</x>
     <y>412</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadPUAServerRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>includeExcludeOptions</receiver>
   <slot>slot_writeSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>606</y>
    </hint>
    <hint type="destinationlabel">
     <x>667</x>
     <y>440</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadPUAScriptRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>includeExcludeOptions</receiver>
   <slot>slot_writeSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>741</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>665</x>
     <y>470</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadPUAAndrRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>includeExcludeOptions</receiver>
   <slot>slot_writeSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>775</x>
     <y>482</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadPUAJavaRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>includeExcludeOptions</receiver>
   <slot>slot_writeSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>487</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>776</x>
     <y>527</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadPUAOsxRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>includeExcludeOptions</receiver>
   <slot>slot_writeSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>388</y>
    </hint>
    <hint type="destinationlabel">
     <x>775</x>
     <y>560</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadPUAToolRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>includeExcludeOptions</receiver>
   <slot>slot_writeSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>441</y>
    </hint>
    <hint type="destinationlabel">
     <x>774</x>
     <y>599</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadPUAUnixRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>includeExcludeOptions</receiver>
   <slot>slot_writeSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>408</x>
     <y>489</y>
    </hint>
    <hint type="destinationlabel">
     <x>777</x>
     <y>631</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadPUAWinRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>includeExcludeOptions</receiver>
   <slot>slot_writeSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>416</x>
     <y>543</y>
    </hint>
    <hint type="destinationlabel">
     <x>761</x>
     <y>667</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_enablePUACheckBoxClicked()</slot>
  <slot>slot_writeSettings()</slot>
 </slots>
</ui>
